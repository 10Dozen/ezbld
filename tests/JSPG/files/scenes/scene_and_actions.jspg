# SceneName

This is a scene.

@ Action 1

This is an first action for SceneName.

@ Action 2

This is an second action for SceneName.
