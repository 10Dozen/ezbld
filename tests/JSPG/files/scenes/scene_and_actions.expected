[0]
cls = JSPGScene
name = SceneName
desc = This is a scene.

[1]
cls = JSPGAction
name = Action 1
scene = SceneName
desc = This is an first action for SceneName.

[2]
cls = JSPGAction
name = Action 2
scene = SceneName
desc = This is an second action for SceneName.
